loader-tests/utils.ts

```diff
@@ -0,0 +1,50 @@
+import pico from 'picocolors'
```

Comment:
We can already use `util.styleText` from [the latest Node.js](https://nodejs.org/en/blog/release/v20.12.0#text-styling)

Reply:
Not sure tho, I'm getting this error if I try to use it w/ `node v20.12.0`.
```
Module '"node:util"' has no exported member 'styleText'.ts(2305)
```

Reply:
@ai , it's probably `@types/node`, last release was 1w ago, and it's on `20.11.30`. It needs to catch up.